README.md

```diff
@@ -48,3 +39,118 @@ The code layout consists mainly of:
    - `transform_*.py` - transform the extracted raw data into output formats
 - `tests/` - the unit tests
 
+## File formats
+This scraper generates three different JSON file types.
+
+### `revisions.json`
+This file contains the revision hashes of the changesets the probe files were scraped. These hashes are mapped to an human-readable version string.
+
+```
+{
+  "<channel>": {
+    "<revision hash>": {
+      "version": "<human-readable version string>"
+    },
+    ...
+  },
+  ...
+  "aurora": {
+    "1196bf3032e1bce1fb07a01fd9082a767426c5fb": {
+      "version": "51"
+    },
+  },
+  ...
+}
+```
+
+### `general.json`
+This file contains general properties related to the scraping process. As of today, it only contains the `lastUpdate` property, which is the day and time the scraping was performed, in ISO 8601 format.
+
+```
+{
+  "lastUpdate": "2018-01-15T17:57:08.944690+01:00"
+}
+```
+
+### Probe data file
+This file contains the data for the probes. The data might be spread across multiple files. It has the following format:
+
+```
+{
+  "<probe type>/<probe name>": {
+    "history": {
+      "<channel>": [
+        {
+          "cpp_guard": <string or null>,
```

Comment:
Could we get a description of this field?

Reply:
Good catch! This probe data comes directly from the Telemetry probe definition. I've added a note at the bottom of this section with a link to the Telemetry docs with the full documentation on all the fields that can be found there.